core/bindings.rs

```diff
@@ -625,6 +627,90 @@ fn decode(
   };
 }
 
+struct SerializeDeserialize {}
+
+impl v8::ValueSerializerImpl for SerializeDeserialize {
+  #[allow(unused_variables)]
```

Comment:
What unused variables?

Reply:
This function is required by `v8::ValueSerializerImpl` but is never used in `core/bindings.rs` (yet I think)